lumicks/pylake/force_calibration/calibration_models.py

```diff
@@ -262,6 +264,17 @@ def _drag(self):
         """
         return self.drag_coeff * self._drag_correction_factor
 
+    def _set_drag(
+        self,
+        drag,
+        new_name="gamma_0_lateral",
```

Comment:
Nitpick: `gamma_0_lateral` implies that we are using `gamma_0` from the lateral force calibration, while we would actually use  `gamma_ex`. I wonder if we shouldn't call it `gamma_ex_lateral`

Reply:
Good point. Updating it.